### ISSUE TITLE
What your current development stack?

### ISSUE THREAD

**Issue by drKraken** [arthurvr/ama#5]

**Comment by arthurvr** [comment]
Any specific pieces of my stack you're interested in? I could probably write whole books about development stacks, tooling and workflows.

**Comment by drKraken** [comment]
I'm more interested in stack for frontend development,you know, libs/ frameworks, workflow especially for js/node development, favorite editor.

**Comment by arthurvr** [comment]
> libs/ frameworks,

JavaScript has the "yet another framework syndrome" and I don't try to fall into that trap. I try out much things but I try to focus on knowing one (or a few) of technologies really good. Currently React is the one I like the most. (and jQuery of course :wink:) 

In the past I was into Backbone and I've been doing fun stuff with Dojo. Kinda moved away from those lately.

Vanilla JavaScript is really underestimated imho.

> workflow especially for js/node development,

Again, any specifics you'd like to hear? I like Gulp and I use browserify, Babel, ESLint, and basically everything Yeoman scaffolds for me.

Besides from that, I :heart: [tiny node modules](https://github.com/sindresorhus/ama/issues/10#issuecomment-117766328).

I use npm for most of my package-management. 

> favorite editor.

I use Vim, in combination with tmux and iTerm. Can't imagine text editors like Sublime anymore. I spent days (weeks, months) learning vim and digging into its documentation. I probably would have better spent my time building actual stuff, but now I have pretty okay vim skills and I really benefit from that.

I use a crazy big line height, combined with the *Tommorow Night Bright* color scheme, the font I use is called *Source Code Pro*, 15px.

-

Thanks for the cool question. Feel free to shoot anything more. Closing this one but if you want anything additional, feel free.
